ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -508,6 +508,46 @@ In AVD 4.0.0, we had "switched" as the default value for `ethernet_interfaces[].
 
 With AVD 5.0.0, the default value for `type` in `ethernet_interfaces` and `port_channel_interfaces` is no longer supported. The `type` key must now be explicitly defined in the input variables if it is needed in the configuration and documentation.
 
+### Data model for BGP additional-paths in `router_bgp.address_family_evpn/_vrfs[].ipv4/_vrfs[].ipv6/_path_selection` has been changed
+
+In AVD 5.0.0, the data models for `router_bgp.address_family_evpn.vrfs[].address_family_ipv4.bgp.additional_paths`, `router_bgp.address_family_evpn.vrf[].address_family_ipv6.bgp.additional_paths`, `router_bgp.address_family_evpn.address_family_evpn.bgp.peer_groups[].additional_paths`, `router_bgp.address_family_evpn.address_family_evpn.bgp_additional_paths`, `router_bgp.address_family_evpn.address_family_path_selection.bgp.additional_paths`, `router_bgp.address_family_evpn.address_family_path_selection.neighbors[].additional_paths` have been changed.
+
+Added support for filtering of BGP additional paths using prefix-lists in `router_bgp.address_family_evpn.vrfs[].address_family_ipv4.bgp.additional_paths` and `router_bgp.address_family_evpn.vrfs[].address_family_ipv6.bgp.additional_paths`.
```

Comment:
new feature should not be added in porting guide

Reply:
Removed the new feature description.